src/main/java/Lesson5/Task1/Main.java

```diff
@@ -0,0 +1,11 @@
+package Lesson5.Task1;
+
+public class Main {
+    public static void main(String[] args) {
+        Zoo zoo = new Zoo();
+        for(int i=0;i<8;i++){
+            zoo.zooList.add(i,new Zoo());
```

Comment:
Add separate method to add zoo (not to the list directly from Main), also It'll be better to add not noname Zoo but add some name: bear, fox etc.